src/DataLib/fendl32B_retrofit/reaction_data.py

```diff
@@ -98,9 +98,12 @@ def nucleon_changes(emission_dict):
             change in neutrons and protons in a nucleus as a result of neutron
             activation and subsequent decay. The array is in the format of
             array([neutron_change, proton_change]).
+
+            If emission_dict is empty, NP_change will be returned as an array
+            of None values, in the format array([None, None]).
     """
     
-    NP_change = array([None , None])
+    NP_change = array([None, None])
```

Comment:
I think this can just be `None` as default:
```suggestion
    NP_change = None
```